XamlControlsGallery/ControlPages/ToggleButtonPage.xaml

```diff
@@ -40,5 +40,18 @@
                                                   Value= "IsEnabled=&quot;False&quot; "/>
             </local:ControlExample.Substitutions>
         </local:ControlExample>
+        <local:ControlExample HeaderText="A ToggleButton with Reveal style applied.">
+            <StackPanel Orientation="Horizontal" VerticalAlignment="Top">
+                <ToggleButton Content="ToggleButton"
+                              Style="{StaticResource ToggleButtonRevealStyle}"
```

Comment:
ToggleButtonRevealStyle looks a little strange. It's like the reveal and the regular hover state are fighting each-other and suddenly snap from one style to the other. Oh, well.

Reply:
Maybe we should chime in on [this issue over at WinUI](https://github.com/microsoft/microsoft-ui-xaml/issues/1126) with this problem. There is already a discussion about the reveal style in general and how to improve it.